goldmane/proto/api.proto

```diff
@@ -61,13 +61,13 @@ message FlowListRequest {
   int64 aggregation_interval = 7;
 }
 
-// FlowListResult is a message containing a FlowResult or ListMetadata (it will only ever contain one or the other).
+// FlowListResult is a message containing a list of FlowResults and ListMetadata.
 message FlowListResult {
-  // ListMetadata specifies information about the filter hints to be returned after this message.
+  // ListMetadata specifies list information about the hints returned.
```

Comment:
```
Meta specifies metadata about the returned flows
```

^ Same here - the field name is "Meta" and the type is "ListMetadata"

Also, this one still mentions "hints" instead of "flows"